src/editor/EditorCommandHandlers.js

```diff
@@ -202,18 +207,26 @@ define(function (require, exports, module) {
         // Are there any non-blank lines that aren't commented out? (We ignore blank lines because
         // some editors like Sublime don't comment them out)
         var i, line, prefix, commentI,
+            lineIndentation = [],
             containsNotLineComment = _containsNotLineComment(editor, startLine, endLine, lineExp);
-        
+
         if (containsNotLineComment) {
             // Comment out - prepend the first prefix to each line
             for (i = startLine; i <= endLine; i++) {
-                editGroup.push({text: prefixes[0], start: {line: i, ch: 0}});
+                if (PreferencesManager.get("indentLineComment")) {
```

Comment:
Thanks. I renamed the pref just before pushing and forgot about this occurence.
Fixed now.